packages/orm/src/BaseEntity.ts

```diff
@@ -86,6 +86,18 @@ export abstract class BaseEntity implements Entity {
     return this.__orm.deleted !== undefined;
   }
 
+  get isDirtyEntity(): boolean {
+    return Object.keys(this.__orm.originalData).length > 0;
+  }
+
+  get isPendingFlush(): boolean {
```

Comment:
I wouldn't mind keeping this a little more internal vs. obviously accessible in the entities public API, but not a huge deal.